<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>탑</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>44</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>서폿</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>44</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>원딜</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>44</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>정글</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>44</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>미드</string>
   </property>
  </widget>
  <widget class="QLabel" name="top_win">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>//</string>
   </property>
  </widget>
  <widget class="QLabel" name="jun_win">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>//</string>
   </property>
  </widget>
  <widget class="QLabel" name="mid_win">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>//</string>
   </property>
  </widget>
  <widget class="QLabel" name="ad_win">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>//</string>
   </property>
  </widget>
  <widget class="QLabel" name="sup_win">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>//</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>54</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1팀</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>54</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>2팀</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="win_1">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>20</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>190</y>
     <width>54</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>VS</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="win_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>190</y>
     <width>20</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>114</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>승리 팀</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="team_win_1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1팀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="team_win_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>2팀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>승률 확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>team_win_1</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>win()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>team_win_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>lose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>per()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>win()</slot>
  <slot>lose()</slot>
  <slot>per()</slot>
 </slots>
</ui>
